mods/ctf/ctf_map/maps_catalog.lua

```diff
@@ -107,19 +93,58 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 	local name = player:get_player_name()
 
 	if fields.btn_prev then
+		indices[name] = indices[name] - 1
 		show_catalog(name, indices[name] - 1)
 	elseif fields.btn_next then
+		indices[name] = indices[name] + 1
 		show_catalog(name, indices[name] + 1)
```

Comment:
```suggestion
		show_catalog(name)
```

The second a parameter is being removed in this PR.

Reply:
Done